docs/csharp/language-reference/compiler-messages/cs7003.md

```diff
@@ -0,0 +1,29 @@
+# Compiler Error CS7003
+Unexpected use of an unbound generic name
+
+This error occurs if you use a generic type needing one parameter type without passing any generic parameter type name between the angle brackets. This use may be a variable declaration, or an object instanciation.
+
+## To correct this error  
+  
```

Comment:
Since there aren't multiple items, this shouldn't be a list.

Reply:
Thanks for this remark @rpetrusha. I just pushed a commit to fix this.